ons_alpha/core/wagtail_hooks.py

```diff
@@ -13,3 +20,33 @@ def register_icons(icons):
     return icons + [
         "wagtailfontawesomesvg/solid/chart-simple.svg",
     ]
+
+
+@hooks.register("after_publish_page", order=float("inf"))
+def notify_slack_of_publish(request, page):
+    if (webhook_url := settings.SLACK_PUBLISH_NOTIFICATION_WEBHOOK_URL) is None:
+        return
+
+    revision = page.latest_revision
+    user = revision.user
```

Comment:
It would be good to also output the release date time if the page has that field, so it can be sanity-checked quickly.